test/controllers/api/v1/api_controller_test.rb

```diff
@@ -149,7 +149,10 @@ def setup
     assert_nil validator_with_all_data["ping_time"]
 
     # IP
```

Comment:
Please add a test for checking if `latitude` and `longitude` return values when they are not nil